source/eventcore/drivers/posix/sockets.d

```diff
@@ -57,6 +58,12 @@ version(FreeBSD) {
 		enum IP_MULTICAST_LOOP  = 11;
 	} else import core.sys.freebsd.netinet.in_ : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
 }
+version(DragonFlyBSD) {
+	static if (__VERSION__ < 2077) {
+		enum IP_ADD_MEMBERSHIP  = 12;
+		enum IP_MULTICAST_LOOP  = 11;
+	} else import core.sys.dragonflybsd.netinet.in_ : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
+}
```

Comment:
There are multiple/repeated implementations for this same enum (in vibe.d / vibe-core.d etc), would be nice if they could be standardized in druntime.